internal/core/testing.go

```diff
@@ -376,12 +379,13 @@ func Test(config *TestConfig) func(t *testing.T) {
 			stderrBuffer := &bytes.Buffer{}
 			_, result, err := Bootstrap(&BootstrapConfig{
 				Args:             args,
-				Commands:         config.Commands,
+				Commands:         config.Commands.Copy(), // Copy commands to ensure they are not modified
```

Comment:
Why are the commands modified now?

Reply:
Configs are loaded during `Bootstrap` and aliases are added to commands